src/lib/journey/callbacks/text-output/text-output.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+  import type { ConfirmationCallback, TextOutputCallback } from '@forgerock/javascript-sdk';
+
+  import Button from '$components/primitives/button/button.svelte';
+
+  import xss from 'xss';
+  export let callback: TextOutputCallback;
+  export let choice: ConfirmationCallback;
+
+  const options = choice.getOptions();
+
+  $: message = callback.getMessage();
+
+  const sanitized = xss(message);
+
+  $: selected = choice.getDefaultOption(); // gets the index position
```

Comment:
Have you pushed your most recent changes? This seems like the older, incorrect version of the component we discussed yesterday.